src/Sylius/Bundle/ShopBundle/EventListener/NonChannelLocaleListener.php

```diff
@@ -57,6 +57,10 @@ public function restrictRequestLocale(GetResponseEvent $event): void
         }
 
         $request = $event->getRequest();
+        if($request->attributes && in_array($request->attributes->get('_route'), ['_wdt', '_profiler'])) {
```

Comment:
Is there any possibility the `$request->attributes` would be false? can it be `null`? I believe it can be an empty `ParameterBag` but then it would be covered with the second condition

Reply:
And missing space after `if` :)

Reply:
Yes, when running the phpspec tests there will not always be any attributes set. Actually it's only in the one test I made :) I'll add the space.